examples/31_minimal/waku.config.ts

```diff
@@ -0,0 +1,9 @@
+/** @type {import('waku/config').Config} */
+export default {
+  middleware: () => [
+    import('waku/middleware/context'),
+    import('waku/middleware/dev-server'),
+    import('waku/middleware/handler'),
+    import('waku/middleware/fallback'),
+  ],
+};
```

Comment:
Is this expected to be a consumer facing api?

it seems a little weird to have all the plugins imported dynamically, but at the same time I can't think of any reasons that it isn't good. It just seems different and I am curious as to why it's like this.

Reply:
customizing the entire `middleware` allows devs full flexibility. it's not for everyone, though. it's for advanced users.

Ah, I think I get what you mean. This file is temporary. When `new_defineEntries` replaces `defiineEntries`, this file will be removed.